devito/core/cpu.py

```diff
@@ -84,6 +95,9 @@ def _normalize_kwargs(cls, **kwargs):
         # Blocking
         o['blockinner'] = oo.pop('blockinner', False)
         o['blocklevels'] = oo.pop('blocklevels', cls.BLOCK_LEVELS)
+        o['blockeager'] = oo.pop('blockeager', cls.BLOCK_EAGER)
+        o['blocklazy'] = oo.pop('blocklazy', not o['blockeager'])
```

Comment:
IS this really needed to have an option and its negated?

Reply:
Probably not. So, I made it this way because then in theory you could have both modes... we would need some extra machinery, but one could have eager blocking for some loops and lazy blocking for other, but yeah, I admit we don't really have use cases ATM, so if you prefer I can drop one